src/bridge/cockpitpackages.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2014 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef COCKPIT_PACKAGES_H_
+#define COCKPIT_PACKAGES_H_
+
+#include <glib.h>
+
+typedef struct _CockpitPackage CockpitPackage;
+typedef struct _CockpitPackages CockpitPackages;
+
+gint cockpit_packages_port;
```

Comment:
This doesn't seem to be used anywhere.  (There is cockpit_bridge_packages_port for the tests.)